source/_docs/load-and-performance-testing.md

```diff
@@ -1,200 +1,75 @@
 ---
 title: Load and Performance Testing
 description: Learn how to monitor internal execution performance of your Pantheon Drupal or WordPress site.
-tags: [performance, cache]
-categories: [platform, cache]
+tags: [performance]
+categories: [performance]
 ---
-We highly recommend load testing a site both prior and post launch to ensure your site is optimally configured.
-
-## Before You Begin
-
-You should:
-
-- [Enable New Relic Pro](/docs/new-relic) to monitor internal execution performance without needing any additional modules or tools.
-- Have access to a command-line environment, preferably with administrative privileges.
-
-<div class="alert alert-info" role="alert">
-<h3 class="info">Note</h3>
-<p><strong>Load testing should only be performed on the Live environment</strong>. Dev has much lower default caching settings than other environments to facilitate iterative development. Test has the exact same configuration as Live, but Test can only have one appserver, while Live can have as many as your plan allows. If disruptive behavior occurs outside of the Live environment, the site may be temporarily disabled to prevent disruption to other customers.</p></div>
-
-## Performance vs. Scalability
-
-There are two things to test for:
-
-1. **Performance**: the response time for an individual request
-2. **Scalability**: the ability to deliver with optimal response time to a larger number of concurrent requests
-
-High-performance is the ability to deliver a page in under a second; scalability is the ability to deliver that page in under a second for many requests. It's important to understand the difference between these two dimensions and that there are trade-offs between performance and scalability.
-
-## Verify Varnish is Working

```

Comment:
Seems like verifying Varnish is working is still important before doing a load test? Maybe this can be more concise?

Reply:
Is this still the case now that Global CDN is in place?